src/lib/init-hooks/core-hooks/context.js

```diff
@@ -0,0 +1,31 @@
+const cls = require('boring-cls');
+const uuid = require('node-uuid');
+
+module.exports = function(BoringInjections) {
+
+  const {
+    boring,
+  } = BoringInjections;
+
+  boring.before('add-hooks', function() {
+
+    const ns = cls.createNamespace('http-request');
+    boring.after('app.use', function(ctx) {
+      if (ctx.name === 'healthy') {
+        boring.app.use('context', function contextMiddleware(req, res, next) {
+
+          ns.run(() => {
+            req.corrId = uuid.v4();
```

Comment:
Setting an unique ID per HTTP request.  This ID will be automatically logged in every log line so we can group all the lines logged per request when doing post mortem / debugging.